orca-core/src/test/groovy/com/netflix/spinnaker/orca/config/UserConfiguredUrlRestrictionsSpec.groovy

```diff
@@ -5,16 +5,21 @@ import spock.lang.Unroll
 
 class UserConfiguredUrlRestrictionsSpec extends Specification {
 
+  // Don't try to actually resolve hosts, and control the result of determining whether a host is localhost or link local
+  def spyOn(UserConfiguredUrlRestrictions subject, isLocalhost = false, isLinkLocal = false) {
```

Comment:
Most of the builder arguments in these tests were to dodge code paths that attempted to resolve names, so that has been mocked out and the non-relevant builder arguments removed from the below tests.  Those code paths are now more exercised than before.